app/models/vote.rb

```diff
@@ -1,2 +1,3 @@
 class Vote < ActsAsVotable::Vote
+  KINDS = %w(knapsack approval)
```

Comment:
Why this distinction in the `Vote` class? Where is it used?

Reply:
@voodoorai2000 it is used in `Budget::Group` for validating the `voting_style` attribute. If I remember correct it doesn't have any functional use in the vote class. Shall I move it to `Budget::Group`?

Reply:
+1 to moving it somewhere else. How about to the `Budget` class itself? 🤔

Reply:
I will move it there and make it `VOTING_STYLES = ...`